airflow/config_templates/config.yml

```diff
@@ -1692,6 +1692,13 @@ webserver:
       type: string
       example: ~
       default: "5 per 40 second"
+    caching_hash_method:
+      description: |
+        The caching algorithm used by the webserver. Must be a valid hashlib function name.
+      version_added:
```

Comment:
I guess we should put 2.7.0 here?

Reply:
Unless @ephraimbuddy will decide to merge in to 2.6.0 ?

Reply:
I know I wrote on slack that it is 2.7 - but on a hindsight - unless it's not too late - that one looks super safe

Reply:
Ah. I see it is missing at all :(

Reply:
Ah whoops! Let me know if I can do a fast follow up here to add 2.7.0 here @potiuk

Reply:
Let's tag it 2.6.0. I will no longer do the RC today

Reply:
Yeah. Can you please add 2.6.0 `added` PR @vchiapaikeo :)?

Reply:
Yes, not a problem. PR: https://github.com/apache/airflow/pull/30688